test/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../tsconfig.base.json",
   "compilerOptions": {
     "tsBuildInfoFile": "../build/tsbuildinfo/test",
-    "types": ["node", "mocha", "flot"]
+    "types": ["node", "mocha", "flot", "@kbn/test/types/ftr_globals"]
```

Comment:
Went ahead and unified the global mocha and snapshot types under this directory in `@kbn/test` and included them in both the `test/tsconfig.json` and `x-pack/test/tsconfig.json` projects.